security/pkg/pki/ca/ca.go

```diff
@@ -176,21 +169,7 @@ func NewIstioCA(opts *IstioCAOptions) (*IstioCA, error) {
 
 // Sign takes a PEM-encoded CSR and returns a signed certificate. If the CA is a multicluster CA,
```

Comment:
Could you update the comments as well? This now can only talks about forCA is true or not.

Reply:
Done.